### DISCUSSION TITLE
.Hex File

### DISCUSSION THREAD

**Discussion by kefkevm** [djgreeb/Sampler-DJS-500#5] [Ideas]
Hi,

Is it possible to make a .Hex file?
Just like your cdj1000mk3 new life project.

greets

**Comment by djgreeb** [comment]
Hi, I have added a hex file to the repository
Sampler-DJS-500/tree/main/Firmware/Sampler/HEX/F7.hex

**Reply by kefkevm** [reply]
did you test this file?
uploading it at the same way as the cdj1000newlife.hex does not work. 
the cdj1000newlife.hex runs fine.
maybe something wrong?
